.gitignore

```diff
@@ -1,5 +1,6 @@
 .DS_Store
 node_modules/
+bin/
```

Comment:
If you take my suggestion to move the script to this folder, then we should not ignore it